Dockerfile.ci

```diff
@@ -1020,273 +1019,31 @@ if [[ ${DOWNGRADE_SQLALCHEMY=} == "true" ]]; then
     pip check
 fi
 
+pip uninstall --root-user-action ignore "pytest-capture-warnings" -y >/dev/null 2>&1 || true
```

Comment:
Small hack for now - I vendored-in pytest-capture-warning and simplified it / got rid of warning summary cluttering the output - this small uninstall makes sure it won't be there in breeze even if it has been cached. 

We will remove it several days after rebase where the cache is rebuil for CI and most people.